playbooks/host_vars/crap.yml

```diff
@@ -3,7 +3,7 @@
 
 passworded_users:
   - username: "fil"
-    password_hash: "{{ lookup('community.general.lastpass', 'fil@crap.siasky.net', field='password') | password_hash('sha512', 65534 | random(seed=inventory_hostname) | string) }}"
+    password_hash: "{{ (lookup('community.general.lastpass', 'fil@crap.siasky.net', field='password') | password_hash('sha512', 65534 | random(seed=inventory_hostname) | string)) if (lastpass_required|default(False)) else None }}"
```

Comment:
nit: is it possible in ansible to define some vars to make this more readable? could the following be a variable to better understand it?
```
(lookup('community.general.lastpass', 'fil@crap.siasky.net', field='password') | password_hash('sha512', 65534 | random(seed=inventory_hostname) | string))
```

Reply:
Created a f/u issue #99